Text/TextAnalyticsForHealth/TextAnalyticsForHealth.cs

```diff
@@ -0,0 +1,189 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using AzureCognitiveSearch.PowerSkills.Common;
+using Azure.AI.TextAnalytics;
+using Azure;
+using System.Globalization;
+
+namespace AzureCognitiveSearch.PowerSkills.Text.TextAnalyticsForHealth
+{
+    public static class TextAnalyticsForHealth
+    {
+        public static readonly string textAnalyticsApiEndpointSetting = "TEXT_ANALYTICS_API_ENDPOINT";
+        public static readonly string defaultTextAnalyticsEndpoint = "https://centralus.api.cognitive.microsoft.com";
```

Comment:
>centralus [](http://example.com/codeflow?start=78&length=9)

Oh, so the default endpoint is regional? Yea perhaps we shouldn't have a default endpoint then, since this will depend on what region the customer creates their resources in with the ARM template

Reply:
Basically what I'm saying is if this is the case, we should just get rid of the default endpoint altogether. Sorry for the confusion and extra work earlier. :P

Reply:
So right now the endpoint will be set in the settings by the ARM template, so if they are deploying it from there it should all be gravy. If they deploy it on their own, or for some reason the ARM template fails, and it causes this setting to not be set, would we rather have the function error out on all requests or use a regional endpoint which is less efficient?

Reply:
The problem isn't that the regional endpoint is less efficient, its that I'm pretty sure it straight up won't work if the region of the key you are using isn't created in Central US. Hence why its proabably just better to require the endpoint in the function settings as you originally did, especially since for 99% of users the ARM template will just take care of it for them anyways.